src/migrations/20200101101824-create-request.js

```diff
@@ -8,9 +8,6 @@ export const up = (queryInterface, Sequelize) => queryInterface.createTable('Req
   requesterId: {
     type: Sequelize.INTEGER
   },
-  tripId: {
```

Comment:
Do not modify existing migration. Instead, create a new migration and alter/drop the column